### DISCUSSION TITLE
Multicluster High Availability

### DISCUSSION THREAD

**Discussion by DaspawnW** [linkerd/linkerd2#5321] [Q&A] [answered]
Hi Linkerd Team,

I was able to setup a Multicluster link as described in your documentation, really good work you folks have done :-)

With this I have two general questions:
1. Is there any reason why we should not be able to scale the gateway as well as the service-mirror to a higher value then the default replica count of 1?
2. We are using for the connection from the left to the right cluster an AWS ELB (aka Classic load balancer). This ELB changes from time to time its ip addresses. Is there any problem with such kind of setup or could it produce some problems as the IP addresses are somewhere hardcoded e.g. in a service?

Thank you in advance,
Best regards,
 Björn

**Comment by grampelberg** [comment, marked as answer]
> Is there any reason why we should not be able to scale the gateway as well as the service-mirror to a higher value then the default replica count of 1?

I would recommend this.

> We are using for the connection from the left to the right cluster an AWS ELB (aka Classic load balancer). This ELB changes from time to time its ip addresses. Is there any problem with such kind of setup or could it produce some problems as the IP addresses are somewhere hardcoded e.g. in a service?

You'll run into normal DNS caching issues but otherwise AFAIK there won't be any issues.

**Comment by DaspawnW** [comment]
Hi @grampelberg,

Thanks for the really fast response :-)

`I would recommend this.` Thats perfect, only asking for a friend, never heard about such stuff that HA is not possible ;-)

`You'll run into normal DNS caching issues but otherwise AFAIK there won't be any issues.`
Sorry can you please explain a little bit more, what does you mean here with DNS caching. There are two points that come to my mind:
* TTL of ELB DNS Entry
* Some application based DNS Caching, worst example is JVM :-D

**Reply by grampelberg** [reply]
It'd be the TTL as the endpoint IPs/DNS will be managed by golang (the service mirror) or rust (the proxy itself).

**Reply by DaspawnW** [reply]
Thanks again 😀
